src/test/java/org/mockitousage/debugging/VerificationListenerCallBackTest.java

```diff
@@ -12,30 +11,22 @@
 import org.mockito.verification.VerificationEvent;
 import org.mockito.verification.VerificationMode;
 
-
 import java.lang.reflect.Method;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.*;
 
 public class VerificationListenerCallBackTest {
-    private Method invocationWanted;
-    private RememberingListener listener;
-    private MockitoFramework mockitoFramework;
-
-    @Before
-    public void setUp() throws NoSuchMethodException {
```

Comment:
But interdependant. Since the _given_ part of each tests is about the framework listner I think this is a good practice to put it in the test method. If this was regular setup that had no impact on the _given_ part of the scenario then it's probably ok in a before method.